docs/concepts/protocol/02-stream-types.mdx

```diff
@@ -143,3 +145,114 @@ accumulate, so if the recipient doesn't withdraw them, they will be able to with
 The advantage of using Unlock in Steps instead of a normal vesting contract is that Sablier automates the entire
 process. No more worries about setting up vesting contracts or creating a user interface for your employees to claim
 their tokens.
+
+### Monthly Unlocks
+
+Monthly Unlocks is a variation of the Unlock in Steps model, with the distinction of having monthly intervals. This
+model is particularly suited for structured yet flexible token distribution plans such as distributing a set salary
```

Comment:
How about we say something simple here (it looks a bit complicated to me):

Monthly Unlocks is a special case of Unlock in Steps where assets are unlocked on the same day every month. This is particularly suited for use cases where users want to distribute tokens every month such as distributing salaries.